integration-common.nix

```diff
@@ -0,0 +1,124 @@
+# This module is the common base for the NixOS and nix-darwin modules.
```

Comment:
This should be moved out of the project root directory. For example, you can put it in the nixos directory instead.

Reply:
Will do in follow-up.